app/views/group/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h3 class="mb-3">Categories</h3>
+
+<% if @groups.count > 0 %>
+  <% @groups.each do |group| %>
+    <%= link_to group.entities(group), class: "card mb-2" do %>
```

Comment:
- [ ] Kindly use the correct syntax to link to a single group
```html.rb
  <%= link_to group_entities_path(group), class: "card mb-2" do %>
```
![image](https://user-images.githubusercontent.com/65068771/211950639-749f319b-0a68-40c6-8412-c5b8f31de2dd.png)

Reply:
done